Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -112,41 +64,74 @@
+            "label": "Who used",
+            "type": 2,
+            "description": "Choose which custom events and page views the users should have sent for this report.",
+            "isRequired": true,
+            "query": "datatable(value:string, text:string, selected: string)[\r\n'pageViews,customEvents,requests', 'Any Custom Events, Request or Page View', '',\r\n'pageViews,customEvents', 'Any Custom Events or Page View', 'true',\r\n'customEvents','Any Custom Event', '',\r\n'pageViews','Any Page View', '',\r\n'requests','Any Request', ''\r\n]",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
           },
           {
             "id": "fd41f402-40ca-4dc5-bd29-38ec93f9d521",
             "version": "KqlParameterItem/1.0",
             "name": "UserEvent",
             "label": "Selected events",
             "type": 2,
-            "description": "Choose multiple events and page views, the report will show users who did at least one of the events or page views, not necessarily all of the selected events and page views.",
+            "description": " Choose multiple events and page views, the report will show users who did at least one of the events or page views, not necessarily all of the selected events, requests and page views.",
             "isRequired": true,
             "multiSelect": true,
             "quote": "'",
             "delimiter": ",",
-            "query": "union {Whoused}\r\n    | where timestamp {TimeRange}\r\n    | project name, group = case(itemType == 'pageView', 'Page Views', 'Custom Events')\r\n    | order by group,name asc\r\n    | summarize count() by group, name\r\n    | project value=name, label=name, group, selected=''",
-            "value": [
-              "value::all"
-            ],
+            "query": "union {Whoused}\r\n    | where timestamp {TimeRange}\r\n    | project name, group = case(itemType == 'pageView', 'Page Views', itemType == 'customEvent', 'Custom Events', 'Requests')\r\n    | summarize count() by group, name\r\n    | project value=name, label=name, group, selected=''\r\n    //| union showCohort\r\n    | order by group asc, label asc",
             "typeSettings": {
               "additionalResourceOptions": [
                 "value::all"
               ],
+              "selectAllValue": "*",
               "showDefault": false
             },
             "queryType": 0,
-            "resourceType": "microsoft.insights/components"
+            "resourceType": "microsoft.insights/components",
+            "value": [
+              "value::all"
+            ]
           },
           {
             "id": "8cd9003c-8e76-4d21-8b22-b7a11197f42f",
             "version": "KqlParameterItem/1.0",
             "name": "ByValue",
-            "label": "By value",
+            "label": "By value x-axis",
```

Comment:
Nit: I wonder if we should put x-axis in parens

Reply:
will talk with PM